orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -91,6 +115,7 @@ def __init__(
         # Inspector for runtime information, for example input data, memory usage, etc.
         self._runtime_inspector = RuntimeInspector(self._logger, self._original_module)
         self._runtime_inspector.memory_ob.enable_memory_stats_by_step(self._runtime_options.print_memory_stat_by_step)
+        self._embedding_module_to_padding_density_map = {}
```

Comment:
i would suggest to put this in runtime_inspector still.

Reply:
yes, moved, thanks